src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,100 @@ title = "Releases"
+  `benchmark` and `verify`, support a `--context` argument to specify the
+  originating context, and a `--tocontext` argument to specify the target
+  context.
+  * `diagnose` operates on all accessible contexts by default, except
+  `diagnose firewall inter-cluster` and `diagnose firewall nat-traversal`
+  which rely on an originating context specified by `--context` and a
+  remote context specified by `--remotecontext`.
+  * Namespace-based commands such as `export` will use the namespace given
+  using `--namespace` (`-n`), if any, or the current namespace in the
+  selected context, if there is one, rather than the `default`
+  namespace.
+  * These commands also support all connection options supported by
+  `kubectl`, so connections can be configured using command arguments
+  instead of kubeconfigs.
+  * Existing options (`--kubecontext` etc.) are preserved for backwards
+  compatibility, but are deprecated and will be removed in the next
+  release.
+
+### Other changes
+
+* The flannel CNI is now properly identified during join.
+* A new ServiceExport status condition type named Synced was added that indicates whether or not the ServiceImport
+was successfully synced to the broker.
+* Service Discovery now handles updates to an exported service and updates/deletes the corresponding ServiceImport accordingly.
+* Deploying using Shipyard now uses specific image overrides determined by the `PRELOAD_IMAGES` variable.
+For developers using `make deploy` (or `make e2e`) there won't be a noticeable change in behavior, but only those images
+specified by the `PRELOAD_IMAGES` will be preloaded and used. Developers can also specify the variable explicitly to further
+control which images get preloaded and used.
+* Added a centralized script for getting `subctl`. It now impersonates `subctl` so that users who run `make shell` will have it
+downloaded the first time they try to use it. To influence the version being installed, the `SUBCTL_VERSION` variable can be set.
+* All `shflags` usage has been removed from Shipyard and consuming projects.
+Users/Developers running commands with `using=` are not affected.
+Users/Developers running commands with `*_ARGS=--flag` should switch to using environment variables instead.
+For example, if you used to run `make deploy DEPLOY_ARGS='--settings ...'`, please run `make deploy SETTINGS='...'` instead.
+* If it’s not given a custom directory, `subctl gather` stores all its output in a directory
+named `submariner-` followed by the current date and time (in UTC) in "YYYYMMDDHHmmss" format.
+* Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
+`clusterset.local`. This prevents unnecessary retries.
+* Cloud cleanup for OpenStack now identifies and deletes failed machinesets.
+* Privileges of the Route Agent and Gateway pods were reduced as they don’t need to access persistentvolumeclaims and secrets.
```

Comment:
```suggestion
* Privileges of the Route Agent and Gateway pods were reduced as they don’t need to access PersistentVolumeClaims and Secrets.
```

https://kubernetes.io/docs/contribute/style/style-guide/#referring-to-kubernetes-api-resources
https://kubernetes.io/docs/concepts/storage/persistent-volumes/

I still find the code style guidance confusing, but for sure they should be UpperCamel.